nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/SetHealthCheckServiceNotifierConfiguration.java

```diff
@@ -0,0 +1,239 @@
+
+import fish.payara.nucleus.healthcheck.HealthCheckService;
+import fish.payara.nucleus.healthcheck.configuration.HealthCheckServiceConfiguration;
+import fish.payara.nucleus.notification.configuration.CDIEventbusNotifier;
+import fish.payara.nucleus.notification.configuration.DatadogNotifier;
+import fish.payara.nucleus.notification.configuration.EmailNotifier;
+import fish.payara.nucleus.notification.configuration.EventbusNotifier;
+import fish.payara.nucleus.notification.configuration.HipchatNotifier;
+import fish.payara.nucleus.notification.configuration.JmsNotifier;
+import fish.payara.nucleus.notification.configuration.NewRelicNotifier;
+import fish.payara.nucleus.notification.configuration.NotificationServiceConfiguration;
+import fish.payara.nucleus.notification.configuration.Notifier;
+import fish.payara.nucleus.notification.configuration.NotifierConfiguration;
+import fish.payara.nucleus.notification.configuration.NotifierConfigurationType;
+import fish.payara.nucleus.notification.configuration.NotifierType;
+import fish.payara.nucleus.notification.configuration.SlackNotifier;
+import fish.payara.nucleus.notification.configuration.SnmpNotifier;
+import fish.payara.nucleus.notification.configuration.XmppNotifier;
+import fish.payara.nucleus.notification.log.LogNotifier;
+
+/**
+ * Admin command to set enabled and noisy flags for a specific {@link Notifier} configuration as part of the
+ * {@link HealthCheckServiceConfiguration}.
+ * 
+ * This should not be confused with the {@link NotifierConfiguration} itself for that specific {@link Notifier} that
+ * exists as part of {@link NotificationServiceConfiguration}.
+ * 
+ * When {@link #noisy} parameter is not specified the according {@link NotifierConfiguration} is loaded from the
+ * {@link NotificationServiceConfiguration} to use its {@link NotifierConfiguration#getNoisy()} as automatic fallback
+ * setting.
+ *
+ * @author jan
+ */
+@Service(name = "set-healthcheck-service-notifier-configuration")
+@PerLookup
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CLUSTERED_INSTANCE, CommandTarget.CONFIG})
+@RestEndpoints({
+        @RestEndpoint(configBean = HealthCheckServiceConfiguration.class,
+                opType = RestEndpoint.OpType.POST,
+                path = "set-healthcheck-service-notifier-configuration",
+                description = "Sets the Configuration for specific Notifier for the HealthCheck Service")
+})
+public class SetHealthCheckServiceNotifierConfiguration implements AdminCommand {
+
+    /**
+     * A list of all "known" {@link Notifier}s, needed to be able to create new child nodes of this type based on the
+     * {@link NotifierType} which is annotated on each of the interfaces.
+     */
+    private static final List<Class<? extends Notifier>> NOTIFIER_TYPES = Arrays.asList(CDIEventbusNotifier.class,
```

Comment:
As this is the only part that requires maintenance when list of actual `Notifier`s changes I tried to get rid of it but I did not find a way to resolve the known subclasses of `Notifier`. I tried `ServiceLocator#getAllServices` but this would always yield empty lists. Any suggestion is welcome.